### DISCUSSION TITLE
Land expansion bonuses (warrior icon)

### DISCUSSION THREAD

**Discussion by IvanSkazkin** [sunflower-land/sunflower-land#1581] [General]
Will there be any bonuses for the very first players in the land expansion? Today I read that it would be impossible to go back to the farm. I have a badge of a warrior and I feel that it does nothing, even worse. I will have 2-3 beds instead of 22 as before. Maybe add nft for first players? After all, we worked, we are the most active. Or x10 resource extraction for a week. Or at least something that will encourage the most active.

I feel like my warrior badge does nothing, even makes me worse. What's the point of me going into land expansion now? Although I am the one who tried and has a badge

**Comment by future2back** [comment]
Thought I was the only one who thought this expansion was too rushed.

I dont have a badge of a warrior but it dont means that i m not active as you.

Land expansion is not encouraging **all** beta testers.  All beta testers have the same importance. We all have things to do on our current farm. It seems they are more concerned with the deadline than how they are going to do it. Id like to know who they're listening to 🎶.
